src/common/KernelBase.hpp

```diff
@@ -308,6 +341,7 @@ class KernelBase
     MPI_Barrier(MPI_COMM_WORLD);
 #endif
     timer.start();
+    CALI_START;
```

Comment:
I put `CALI_START` after and `CALI_STOP` before the existing timers. Is this acceptable? I assume if someone turns caliper on they would be looking at those timings so they should be more accurate and in the opposite case caliper would be off so it shouldn't impact the original timers that much. Not sure how much impact this has.

Reply:
I think that choice makes most sense.

Reply:
If Caliper is off, do the macros become no-ops with a simple run time conditional evaluation?

Reply:
Yes. If RAJAPerf is not built with caliper, `RAJA_PERFSUITE_USE_CALIPER=OFF`, the macros will be defined as [here](https://github.com/LLNL/RAJAPerf/pull/331/files#diff-f1b7ede2b59bf264efbe43bc92349deb28c8f49dc87efa11b4882ab6805beb24R61), so the only operation will be the `#if defined(RAJA_PERFSUITE_USE_CALIPER)`.